Minor wording changes (#214)
gamemanual0/gm0: source/docs/know-your-lingo.rst

===============

Many teams may be unfamiliar with the various terms and slang that are often thrown around at competitions and meets. Here we have compiled a guide of some commonly used terms to get you up to speed.

> For a more comprehensive list of terms, check out the `Glossary <https://gm0.org/en/latest/docs/appendix/glossary.html/>` located in the Appendix.

Reviewer: ```suggestion
For a more comprehensive list of terms, check out the :doc:`/docs/appendix/glossary` located in the Appendix.
```

League Meet
   Low-stress tournament with matches counting towards league rank. No judging.
League Championship

>    Tournament with ranked matches and alliance elimination matches. Judging included. Will advance to States/Regional level. Only for teams participating in that League. 

Reviewer: ```suggestion
   Tournament with ranked matches and alliance elimination matches. Judging included. Will advance to States/Regional level. Only for teams participating in that League.
```

Linear Extension/Slide
   Mechanism responsible for extending parts beyond the starting configuration
   of the robot in a linear fashion.
Arm

>    Mechanism responsible for extending the reach of the robot through rotational motion or multi-axis movement. 

Reviewer: ```suggestion
   Mechanism responsible for extending the reach of the robot through rotational motion or multi-axis movement.
```
